plugin/latexlivepreview.vim

```diff
@@ -184,14 +184,23 @@ EEOOFF
                         \ l:tmp_root_dir . '/' . bib_fn)                                " TODO: may fail if same bibfile names in different dirs
         endfor
 
+        if exists('g:livepreview_biber') && g:livepreview_biber > 0
+            let s:bibexec = 'biber *.bcf'
+        else
+            let s:bibexec = 'bibtex *.aux'
+        endif
+
         " Update compile command with bibliography
         let b:livepreview_buf_data['run_cmd'] =
                 \       'env ' .
                 \               'TEXMFOUTPUT=' . l:tmp_root_dir . ' ' .
                 \               'TEXINPUTS=' . l:tmp_root_dir
                 \                            . ':' . b:livepreview_buf_data['root_dir']
                 \                            . ': ' .
-                \       'bibtex ' . l:tmp_root_dir . '/*.aux' .
+                \ ' && ' .
+                \       b:livepreview_buf_data['run_cmd'] .
+                \ ' && ' .
+                \       'cd ' . l:tmp_root_dir . ' && ' . s:bibexec .
```

Comment:
We should try to avoid using cd as much as possible, as this could lead some unintended behaviors and makes the code quite hard to reason about

Reply:
Agree. It's bitten me more than once on other scripts.

Reply:
Eventually, I wasn't able to find a way around `cd` like behaviour with `bibtex` because it doesn't allow writing to external directories without a special option set outside the purview of vim-llp.